rails-kickoff-template.rb

```diff
@@ -259,85 +259,169 @@ def set_sentry_context
+    ```
+    WEB_CONCURRENCY - Number of Puma workers
+    RAILS_MAX_THREADS - Number of threads per Puma worker
+    SIDEKIQ_CONCURRENCY - Number of Sidekiq workers
+    ```
 
-    ### Email
+    [rack-timeout][rt]:
 
-    _TODO_
+    ```
+    RACK_TIMEOUT_SERVICE_TIMEOUT
+    RACK_TIMEOUT_WAIT_TIMEOUT
+    RACK_TIMEOUT_WAIT_OVERTIME
+    RACK_TIMEOUT_SERVICE_PAST_WAIT
+    ```
 
-    ### Coding Style
+    [rt]: https://github.com/sharpstone/rack-timeout#configuring
 
-    This projects uses RuboCop and ESLint to catch errors and keep style consistent.
+    ### Important rake tasks
 
-    It also uses [lefthook][lh] to manage git hooks. Use `git commit --no-verify` to skip checks, or see [./lefthook.yml](./lefthook.yml) for info on how to change its setup.
+    !TODO: Document important rake taskss
 
-    [lh]: https://github.com/Arkweid/lefthook
+    ### Scheduled tasks
 
+    !TODO: Document any scheduled tasks
 
-    ### Important rake tasks
+    # Deployments
+        
+    ## Staging
 
-    _TODO_
+    [Staging](https://dashboard.heroku.com/apps/!TODO) is automatically 
+    deployed from the `main` branch.
 
-    ### Scheduled tasks
+    ## Production
 
-    _TODO_
+    [Production](https://dashboard.heroku.com/apps/TODO) can be
+    [deployed manually through Heroku's interface](https://dashboard.heroku.com/apps/TODO/deploy/github),
+    or [deployed manually with Git](https://devcenter.heroku.com/articles/git):
 
-    ### Important ENV variables
+    1. Add the appropriate git remotes for staging and/or production deployments:
 
-    Note that this project uses [dotenv](https://github.com/bkeepers/dotenv) to load `.env` files. Use `.env.development` and `.env.test` to setup _shared_ ENV variables for development and test, and use `.env` files ending in `.local` for variables specific to you.
+      ```sh
+      heroku git:remote -r staging -a !TODO-staging
+      heroku git:remote -r production -a !TODO-production
+      ```
 
-    Configuring Servers:
+    2. Push your main branch to production:
 
-    ```
-    WEB_CONCURRENCY - Number of Puma workers
-    RAILS_MAX_THREADS - Number of threads per Puma worker
-    SIDEKIQ_CONCURRENCY - Number of Sidekiq workers
-    ```
+      ```sh
+      git push production main
+      ```
 
-    [rack-timeout][rt]:
+    ## jemalloc
 
-    ```
-    RACK_TIMEOUT_SERVICE_TIMEOUT
-    RACK_TIMEOUT_WAIT_TIMEOUT
-    RACK_TIMEOUT_WAIT_OVERTIME
-    RACK_TIMEOUT_SERVICE_PAST_WAIT
+    This project is served from Heroku. It uses jemalloc to more efficiently
+    allocate memory. You must run the following to setup jemalloc:
+
+    ```bash
+    heroku buildpacks:add --index 1 https://github.com/mojodna/heroku-buildpack-jemalloc.git
     ```
 
-    [rt]: https://github.com/sharpstone/rack-timeout#configuring
+    # Contributing
+
+    ## Branch naming conventions
+
+    Branch names should follow the format: `<ticket_number>-<brief-description>`
```

Comment:
i find it's easier for me to leverage my terminal's tab-completion with this format, what do you think?

```suggestion
    Branch names should follow the format: `<brief-description>-<ticket_number>`
```

Reply:
Sounds good to me - I'll update. I only went with the recommendation because I thought that was the preferred naming, not necessarily because of any personal preference.